routes/_app.tsx

```diff
@@ -25,10 +25,10 @@ export default async function App(
           href="https://esm.sh/reactflow@11.8.2/dist/base.css"
         />
         <link rel="stylesheet" type="text/css" href="/style.css" />
-        <link
-          rel="stylesheet"
-          href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/dark.min.css"
-        />
+        {/*<link*/}
```

Comment:
Hrm...what's up with commenting this out? I'm guessing this will break the highlight color scheme that peek uses. Also, feel free to switch to that dark-violet.min.css here if it makes peek look better or more like the designs.

Reply:
Or, if your styles below make both peek and json look better, then feel free to delete this.

Reply:
Actually we'll probably want to keep this because we don't know what kind of data peek might send us.

Reply:
Whoops, typo on my part. Shouldn't be commented out--missed going back when I was changing the styles. Thanks for catching!